.github/update-release-branch.py

```diff
@@ -88,6 +88,7 @@ def open_pr(
   body.append('')
   body.append('Please review the following:')
   if len(conflicted_files) > 0:
+    body.append('- [ ] The `package.json` file contains the correct version.')
```

Comment:
For consistency with other lines.

```suggestion
    body.append(' - [ ] The `package.json` file contains the correct version.')
```